diff --git a/ldlidar_component/CMakeLists.txt b/ldlidar_component/CMakeLists.txt
index b0f6678..1bae2c2 100644
--- a/ldlidar_component/CMakeLists.txt
+++ b/ldlidar_component/CMakeLists.txt
@@ -50,7 +50,7 @@ if(DEFINED ENV{ROS_DISTRO})
   message(STATUS "* Found ROS2 ${FOUND_ROS2_DISTRO}")
 else()
   message(WARNING "* ROS2 distro variable not set. Trying to figure it out...")
-  set(ROS2_DISTROS "ardent;crystal;dashing;eloquent;foxy;galactic;humble;iron;jazzy;rolling")
+  set(ROS2_DISTROS "ardent;crystal;dashing;eloquent;foxy;galactic;humble;iron;jazzy;rolling;kilted")
   set(ROS2_FOUND FALSE)
   foreach(distro ${ROS2_DISTROS})
     if(NOT ROS2_FOUND)
@@ -74,6 +74,9 @@ if(ROS2_FOUND)
   #elseif(${FOUND_ROS2_DISTRO} STREQUAL "rolling")
   #  message(STATUS "* ROS2 ${FOUND_ROS2_DISTRO} is officially supported by this package.")
   #  add_definitions(-DFOUND_ROLLING)
+  elseif(${FOUND_ROS2_DISTRO} STREQUAL "kilted")
+    message(STATUS "* ROS2 ${FOUND_ROS2_DISTRO} is officially supported by this package.")
+    add_definitions(-DFOUND_KILTED)
   else()
     message(FATAL_ERROR "ROS2 ${FOUND_ROS2_DISTRO} is not officially supported by this package. Correct working is not guarantee.")
   endif()
diff --git a/ldlidar_node/CMakeLists.txt b/ldlidar_node/CMakeLists.txt
index fffca7f..029511f 100644
--- a/ldlidar_node/CMakeLists.txt
+++ b/ldlidar_node/CMakeLists.txt
@@ -15,7 +15,7 @@ if(DEFINED ENV{ROS_DISTRO})
   message(STATUS "* Found ROS2 ${FOUND_ROS2_DISTRO}")
 else()
   message(WARNING "* ROS2 distro variable not set. Trying to figure it out...")
-  set(ROS2_DISTROS "ardent;crystal;dashing;eloquent;foxy;galactic;humble;iron;jazzy;rolling")
+  set(ROS2_DISTROS "ardent;crystal;dashing;eloquent;foxy;galactic;humble;iron;jazzy;rolling;kilted")
   set(ROS2_FOUND FALSE)
   foreach(distro ${ROS2_DISTROS})
     if(NOT ROS2_FOUND)
@@ -39,6 +39,9 @@ if(ROS2_FOUND)
   #elseif(${FOUND_ROS2_DISTRO} STREQUAL "rolling")
   #  message(STATUS "* ROS2 ${FOUND_ROS2_DISTRO} is officially supported by this package.")
   #  add_definitions(-DFOUND_ROLLING)
+  elseif(${FOUND_ROS2_DISTRO} STREQUAL "kilted")
+    message(STATUS "* ROS2 ${FOUND_ROS2_DISTRO} is officially supported by this package.")
+    add_definitions(-DFOUND_KILTED)
   else()
     message(FATAL_ERROR "ROS2 ${FOUND_ROS2_DISTRO} is not officially supported by this package. Correct working is not guarantee.")
   endif()
diff --git a/ldlidar_node/params/ldlidar.yaml b/ldlidar_node/params/ldlidar.yaml
index 8816728..ba33d1a 100644
--- a/ldlidar_node/params/ldlidar.yaml
+++ b/ldlidar_node/params/ldlidar.yaml
@@ -8,7 +8,7 @@
       debug_mode: false # true to activate debug messages

     comm:
-      serial_port: '/dev/ldlidar' # serial port name
+      serial_port: '/dev/ttyAMA0' # serial port name
       baudrate: 230400 # serial baudrate for LDLiDAR_LD19: 230400. Serial baudrate for LDLiDAR_STL27L: 921600
       timeout_msec: 1000 # data communication timeout in msec
